examples/plot_bcic_iv_4_ecog_cropped.py

```diff
@@ -0,0 +1,381 @@
+)
+set_log_level(verbose='WARNING')
+
+# Model training for a specified number of epochs. `y` is None as it is already supplied
+# in the dataset.
+regressor.fit(train_set, y=None, epochs=n_epochs)
+
+# We compute predictions for each finger flexion
+preds_test, y_test = regressor.predict_trials(test_set, return_targets=True)
+
+preds_test = np.pad(preds_test,
+                    ((0, 0), (0, 0), (y_test.shape[2] - preds_test.shape[2], 0)),
+                    'constant',
+                    constant_values=0)
+
+mask = ~np.isnan(y_test[0, 0, :])
+preds_test = np.squeeze(preds_test[..., mask], 0)
+y_test = np.squeeze(y_test[..., mask], 0)
+
+
+######################################################################
+# Plot Results
+# ------------
+######################################################################
+import matplotlib.pyplot as plt
+from matplotlib.lines import Line2D
+import pandas as pd
+
+# Now we use the history stored by Skorch throughout training to plot
+# accuracy and loss curves.
+
+# Extract loss and accuracy values for plotting from history object
+results_columns = ['train_loss', 'valid_loss', 'r2_train', 'r2_valid']
+df = pd.DataFrame(regressor.history[:, results_columns], columns=results_columns,
+                  index=regressor.history[:, 'epoch'])
+
+fig, ax1 = plt.subplots(figsize=(8, 3))
+df.loc[:, ['train_loss', 'valid_loss']].plot(
+    ax=ax1, style=['-', ':'], marker='o', color='tab:blue', legend=False, fontsize=14)
+
+ax1.tick_params(axis='y', labelcolor='tab:blue', labelsize=14)
+ax1.set_ylabel("Loss", color='tab:blue', fontsize=14)
+
+ax2 = ax1.twinx()  # instantiate a second axes that shares the same x-axis
+
+df.loc[:, ['r2_train', 'r2_valid']].plot(
+    ax=ax2, style=['-', ':'], marker='o', color='tab:red', legend=False)
+ax2.tick_params(axis='y', labelcolor='tab:red', labelsize=14)
+ax2.set_ylabel("Pearson correlation coefficient", color='tab:red', fontsize=14)
+ax1.set_xlabel("Epoch", fontsize=14)
+
+# where some data has already been plotted to ax
+handles = []
+handles.append(Line2D([0], [0], color='black', linewidth=1, linestyle='-',
+                      label='Train'))
+handles.append(Line2D([0], [0], color='black', linewidth=1, linestyle=':',
+                      label='Valid'))
+plt.legend(handles, [h.get_label() for h in handles], fontsize=14, loc='center right')
+plt.tight_layout()
+
+# We can compute correlation coefficients for each finger
+corr_coeffs = []
+for dim in range(y_test.shape[0]):
+    corr_coeffs.append(
+        np.corrcoef(preds_test[dim, :], y_test[dim, :])[0, 1]
+    )
+print('Correlation coefficient for each dimension: ', corr_coeffs)
+
+# We plot predicted and target finger flexion
+x_time = np.linspace(0, preds_test.shape[1] / sfreq, preds_test.shape[1])
+fig, ax1 = plt.subplots(figsize=(8, 3))
+ax1.plot(x_time, y_test[0, :], label='Target')
+ax1.plot(x_time, preds_test[0, :], label='Predicted')
+ax1.xlabel('Time [s]')
+ax1.ylabel('Finger flexion')
```

Comment:
```suggestion
ax1.set_xlabel('Time [s]')
ax1.set_ylabel('Finger flexion')
```

needs to be corrected @sliwy